h2o-core/src/main/java/hex/grid/HyperSpaceWalker.java

```diff
@@ -293,7 +296,7 @@ public MP getParams() {
       return _paramsBuilderFactory;
     }
 
-    protected MP getModelParams(MP params, Object[] hyperParams) {
+    public MP getModelParams(MP params, Object[] hyperParams) {
```

Comment:
fiy @Pscheidl is making changes in the grid search API right now

Reply:
I'll do the merge on my own or help with the merge if required, whichever comes first. There is a small refactor which actually extracts all the walkers out of the interface class (otherwise it would get really big and messy). Nothing else I can think of that touches this PR.

Reply:
@michalkurka thanks, I know that @Pscheidl is working on the parallelisation of RGS. I had taken a look at his changes before starting to this PR. I think that merge should not be too difficult. But yes, we are working "close to each other" and should take it into account.